container/apptabs.go

```diff
@@ -330,13 +330,19 @@ func (r *appTabsRenderer) buildOverflowTabsButton() (overflow *widget.Button) {
 			index := i // capture
 			// FIXME MenuItem doesn't support icons (#1752)
 			// FIXME MenuItem can't show if it is the currently selected tab (#1753)
-			items = append(items, fyne.NewMenuItem(r.appTabs.Items[i].Text, func() {
+			ti := r.appTabs.Items[i]
+			mi := fyne.NewMenuItem(ti.Text, func() {
 				r.appTabs.SelectIndex(index)
 				if r.appTabs.popUpMenu != nil {
 					r.appTabs.popUpMenu.Hide()
 					r.appTabs.popUpMenu = nil
 				}
-			}))
+			})
+			// Fixes #4935
```

Comment:
Don’t add comments referring fixed issues, please.
They have no value.

Reply:
Thank you for the quick review. I removed the comment.